concepts/numbers/about.md

```diff
@@ -1,3 +1,107 @@
 # About
 
-TODO: add information on numbers concept
+There are two different kinds of numbers in JavaScript - numbers and bigints
+
+Numbers are the most used, and represent numeric data type in the double-precision 64-bit floating point format.
+
+- `number`: a numeric data type in the double-precision 64-bit floating point format (IEEE 754).
+  Examples are `-6`, `-2.4`, `0`, `0.1`, `1`, `3.14`, `16.984025`, `25`, `976`, `1024.0` and `500000`.
+- `bigint`: a numeric data type that can represent _integers_ in the arbitrary precision format.
+  Examples are `-12n`, `0n`, `4n`, and `9007199254740991n`.
+
+```javascript
+let numericValue = 42;
+// => 42
+```
+
+A number literal like `42` in JavaScript code is a floating-point value, not an integer.
+There is no separate integer type in common everyday use.
+The `BigInt` type was not designed to replace `Number` for everyday uses.
```

Comment:
Is it cased like this or cased in the lower letters, as stated further above?

Reply:
This is the prototype. The lowercase ones are then the primitive type. It's confusing, I'll try to make it less so.